fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -890,25 +889,15 @@ default boolean recordExists(@Nonnull final Tuple primaryKey, @Nonnull final Iso
 
     /**
      * Get the primary key portion of an index entry.
+     * This API is deprecated. Should use {@link Index#getEntryPrimaryKey(Tuple)} instead.
      * @param index the index associated with this entry
      * @param entry the index entry
      * @return the primary key extracted from the entry
      */
+    @API(API.Status.DEPRECATED)
     @Nonnull
     static Tuple indexEntryPrimaryKey(@Nonnull Index index, @Nonnull Tuple entry) {
```

Comment:
Is there a reason this isn't `@Deprecated` in addition to be `@API(API.Status.DEPRECATED)`?

Reply:
Done.